autoload/codefmt.vim

```diff
@@ -454,3 +516,17 @@ endfunction
 function! codefmt#SetWhetherToPerformIsAvailableChecksForTesting(enable) abort
   let s:check_formatters_available = a:enable
 endfunction
+
+""
+" @private
+" Returns all formatters, for debugging.
```

Comment:
Not sure if we want to keep these two around. I found them essential for debugging the tests, though.